fern/commands/config.js

```diff
@@ -0,0 +1,84 @@
+const Listr = require("listr");
+const process = require("process");
+const fs = require("fs");
+const path = require("path");
+
+const MOZCONFIG_PATH = "mozconfig";
+
+module.exports = (program) => {
+  program
+    .command("config")
+    .description("Creates a mozconfig file based on brand and platform")
+    .option(
+      "-b, --brand <BRAND>",
+      "Specify which brand to use"
+    )
+    .option(
+      "-p, --platform <platform>",
+      "Specify which platform to use"
```

Comment:
Can you please add examples of values expected here?
```suggestion
      "Specify which platform to use (e.g. win64, linux, macosx)"
```